Dockerfile

```diff
@@ -49,13 +49,5 @@ COPY ./docker/server.xml /opt/bitnami/tomcat/conf
 RUN chmod 664 /opt/bitnami/tomcat/conf/server.xml
 
 WORKDIR /opt/bitnami/tomcat/lib
-# org.drizzle.jdbc.DrizzleDriver is used by default for both the all tenants and demo tenant DB DataSource
-RUN wget https://repo1.maven.org/maven2/org/drizzle/jdbc/drizzle-jdbc/1.4/drizzle-jdbc-1.4.jar
```

Comment:
this is required, both because it's the current default, and because it is part of Fineract - we can't have it as a dependency in Gradle, but then not have it in the Dockerfile... because the container image that is built by this Dockerfile is distributed by the ASF, I'm assuming we're still bound by LEGAL-462.

Reply:
I've reverted to drizzle.
Mysql 8 won't work with drizzle though, but I guess that's fine for now.
Migrating to mysql v8 in the future should be alot easier with few db tweaks I've done.